packages/utilities/src/useFocusRects.ts

```diff
@@ -56,12 +56,14 @@ export const FocusRectsProvider = FocusRectsContext.Provider;
 /**
  * Initializes the logic which:
  *
- * 1. Subscribes keydown and mousedown events. (It will only do it once per window,
- *    so it's safe to call this method multiple times.)
- * 2. When the user presses directional keyboard keys, adds the 'ms-Fabric--isFocusVisible' classname
- *    to the document body, removes the 'ms-Fabric-isFocusHidden' classname.
- * 3. When the user clicks a mouse button, adds the 'ms-Fabric-isFocusHidden' classname to the
- *    document body, removes the 'ms-Fabric--isFocusVisible' classname.
+ * 1. Subscribes keydown and mousedown events. (It will only do it once for the outmost ThemeProvider/Fabric element in
```

Comment:
>keydown and mousedown events [](http://example.com/codeflow?start=17&length=28)

NIT: this should be updated to reflect all the events that are being subscribed to

Reply:
Updated!